cypress/pages/v2/wizard/page.ts

```diff
@@ -63,7 +63,7 @@ export class Wizard extends PageOptions implements Page {
 
     // kubeadm does not have additional steps
     if (provider === Provider.kubeadm) {
-      this.Buttons.create.click({force: true});
```

Comment:
Actually, we do not really want to click multiple buttons here. Code should be modified to click only a single button that is a `Create cluster`.

Reply:
so maybe i use an id for the button inside the km-button component

Reply:
and also is it like after the click the button already will be disabled 
i add that because the button is actually inside the component i'm targeting